backends/core/models.py

```diff
@@ -74,6 +83,21 @@ def recipients(self):
       return self.emails_for_notifications.split()
     return []
 
+  def _add_multi_to_memcache(self, mapping,retries = 10, time = MEMCACHE_DEFAULT_EXPIRATION_TIME):
+    """
+        Set multiple values in memcache, prefixed by the id of the pipeline
+        and having a default expiration time of 24 hours
+    """
```

Comment:
You can reformat the comment like this:

"""Set multiple values in the cache.

Arguments:
    mapping: Dictionary of key/value pairs to push into the cache.
    max_retries: Number of times to retry setting values into the cache before raising an exception.
    expiration_time: Integer representing the values expiration time in seconds. Defaults to 24 hours.
"""

Feel free to edit these description of course.

Reply:
Done